app/src/main/java/com/example/android/picover/presentation/navigation/NavigationType.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.android.picover.presentation.navigation
+
+import androidx.compose.material3.windowsizeclass.WindowSizeClass
+import androidx.compose.material3.windowsizeclass.WindowWidthSizeClass
+
+enum class NavigationType {
+    NAVIGATION_BAR,
+    NAVIGATION_RAIL,
+    NAVIGATION_DRAWER;
+
+    companion object {
+        fun fromWindowSize(windowSize: WindowSizeClass) = when (windowSize.widthSizeClass) {
+            WindowWidthSizeClass.Compact -> NAVIGATION_BAR
```

Comment:
Redundant branch because `NAVIGATION_BAR` is in `else` block.

Reply:
Done